examples/decomposition/plot_pca_vs_fa_model_selection.py

```diff
@@ -23,7 +23,6 @@
 by Thomas P. Minka is also compared.
 
 """
-print(__doc__)
```

Comment:
Convention is for all examples to ``print(__doc__)`` at the beginning. Please add this back